include/roboteam_ai/bt/Composite.hpp

```diff
@@ -3,9 +3,23 @@
 #include "Node.hpp"
 
 namespace bt {
+    /**
+     * a type alias to create a vector of shared ptrs to nodes
+     */
+    using nvector = std::vector<std::shared_ptr<bt::Node>>;
```

Comment:
I'm not sure if you want this in your header file. It will extend to a lot of other classes and i'm not sure If you want  'nvector' to be used everywhere.